script/lib/copy-assets.js

```diff
@@ -13,7 +13,7 @@ module.exports = function() {
   console.log(`Copying assets to ${CONFIG.intermediateAppPath}`);
   let srcPaths = [
     path.join(CONFIG.repositoryRootPath, 'benchmarks', 'benchmark-runner.js'),
-    path.join(CONFIG.repositoryRootPath, 'dot-atom'),
+    path.join(CONFIG.repositoryRootPath, 'templates'),
```

Comment:
```suggestion
    path.join(CONFIG.repositoryRootPath, 'dot-atom'),
```
Revert this line to coincide with reverting templates -> dot-atom